src/vue/src/components/journal/JournalEndCard.vue

```diff
@@ -3,32 +3,73 @@
         <template slot="header">
             <slot name="edit-button"/>
             <h2 class="theme-h2">
-                End of assignment
+                {{ title }}
             </h2>
         </template>
 
-        <template v-if="journal">
-            <load-wrapper
-                name="Journal assessment end-card"
-                :loading="loading"
-            >
+        <load-wrapper
+            :loading="loading"
+            name="Journal assessment end-card"
+        >
+            <template v-if="!loading">
                 <assessment
-                    v-if="assessment && !loading"
+                    v-if="journal && assessment"
                     :edit="false"
                     :assessment="assessment"
                     :journal="journal"
                 />
-            </load-wrapper>
-        </template>
+                <template v-else>
+                    <theme-select
+                        v-if="rubrics.length > 1"
+                        v-model="selectedRubric"
+                        label="name"
+                        trackBy="id"
+                        :options="rubrics"
+                        :multiple="false"
+                        :searchable="true"
+                        placeholder="Select a rubric"
+                    />
+
+                    <rubric
+                        :rubric="selectedRubric ? selectedRubric : {criteria: []}"
+                        class="mt-2 mb-1"
+                    >
+                        <template slot="tbody:post-criteria">
+                            <rubric-automatic-criterion-display
+                                :label="'Entry points'"
+                                :description="'Sum of all points obtained through journal entries.'"
+                                :value="journal ? journal.entry_points : '-'"
+                            />
+
+                            <rubric-automatic-criterion-display
+                                :label="'Bonus points'"
+                                :description="'Bonus points given to this journal.'"
+                                :value="journal ? journal.bonus_points : '-'"
+                            />
+                        </template>
+                    </rubric>
+                </template>
+            </template>
+        </load-wrapper>
 
-        <deadline-date-display :subject="assignment"/>
+        <deadline-date-display
+            :subject="assignment"
+            class="mr-2"
```

Comment:
```suggestion
            class="mr-2 mt-2"
```

The gap between the rubric and this line is too small.